src/components/nav-bar/NavBar.tsx

```diff
@@ -0,0 +1,55 @@
+import classes from './NavBar.module.scss';
+
+import { GearSix, HouseSimple, Plus } from '@phosphor-icons/react';
+import { NavLink, useLocation } from 'react-router-dom';
+import { Paths } from '../../main';
+
+enum IconWeight {
+	ACTIVE = 'fill',
+	NOT_ACTIVE = 'light',
+}
+
+const BUTTON_ICON_WEIGHT = 'light';
+const ICON_SIZE = 32;
+
+const NavBar = () => {
+	const location = useLocation();
+
+	const getIconWeight = (isActive: boolean): IconWeight => {
+		return isActive ? IconWeight.ACTIVE : IconWeight.NOT_ACTIVE;
+	};
```

Comment:
To avoid overloading this function, since it's based on constants, you should declare it outside of the component.
Every time the component is called this function will be re-defined over and over, and unless it's based on props, it's no good.